lib/cli.js

```diff
@@ -1,19 +1,11 @@
 #!/usr/bin/env node
+const parsedArgv = require('minimist')(process.argv.slice(2));
 
-var install = require('./install');
-var command = require('./command');
-var parsedArgv = require('minimist')(process.argv.slice(2));
+const install = require('./install');
+const runExecutable = require('./run-executable');
 
 if (parsedArgv._[0] === 'install') {
   install(parsedArgv);
 } else {
-  var options = {
-    args: process.argv.slice(2)
-  };
-
-  if (parsedArgv.logLevel) {
-    options.logLevel = parsedArgv.logLevel;
-  }
-
-  command(options);
+  runExecutable(process.argv.slice(2), process.cwd, parsedArgv.logLevel);
```

Comment:
Actually -- is this file even needed? Is it worth providing a CLI for a CLI? The `postinstall` and `postversion` uses in package.json could just be calling the `install` script directly.

Reply:
I'm not sure about the context for this decision, but the README had stated that it could be used for NPM scripts, which made sense to me. Do you think it's worth keeping?

cc @macdonaldr93 @NathanPJF